modules/ui/src/main/scala/lucuma/ui/implicits/package.scala

```diff
@@ -1,40 +1,41 @@
 // Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-package lucuma.ui
+package lucuma.ui.implicits
 
 import cats.Monad
+import cats.Monoid
 import crystal.ViewF
 import crystal.ViewOptF
 import crystal.react.reuse._
+import japgolly.scalajs.react.React
+import japgolly.scalajs.react.vdom.html_<^.*
 import lucuma.core.optics.SplitEpi
 import react.common.EnumValue
 
 import scala.annotation.targetName
 import scala.scalajs.js
 
-package object implicits {
-  implicit class ViewFOpticOps[F[_], A](val self: ViewF[F, A]) extends AnyVal {
-    def zoomSplitEpi[B](splitEpi: SplitEpi[A, B]): ViewF[F, B] =
-      self.zoom(splitEpi.get)(splitEpi.modify)
-  }
+extension [F[_], A](self: ViewF[F, A])
+  def zoomSplitEpi[B](splitEpi: SplitEpi[A, B]): ViewF[F, B] =
+    self.zoom(splitEpi.get)(splitEpi.modify)
 
-  implicit class ViewOptFOpticOps[F[_], A](val self: ViewOptF[F, A]) extends AnyVal {
-    def zoomSplitEpi[B](splitEpi: SplitEpi[A, B]): ViewOptF[F, B] =
-      self.zoom(splitEpi.get)(splitEpi.modify)
-  }
+extension [F[_], A](self: ViewOptF[F, A])
+  def zoomSplitEpi[B](splitEpi: SplitEpi[A, B]): ViewOptF[F, B] =
+    self.zoom(splitEpi.get)(splitEpi.modify)
 
-  implicit class ReuseViewFOpticOps[F[_], A](val self: Reuse[ViewF[F, A]]) extends AnyVal {
-    def zoomSplitEpi[B](splitEpi: SplitEpi[A, B])(implicit ev: Monad[F]): Reuse[ViewF[F, B]] =
-      self.zoom(splitEpi.get)(splitEpi.modify)
-  }
+extension [F[_], A](self: Reuse[ViewF[F, A]])
+  def zoomSplitEpi[B](splitEpi: SplitEpi[A, B])(implicit ev: Monad[F]): Reuse[ViewF[F, B]] =
+    self.zoom(splitEpi.get)(splitEpi.modify)
 
-  implicit class ReuseViewOptFOpticOps[F[_], A](val self: Reuse[ViewOptF[F, A]]) extends AnyVal {
-    def zoomSplitEpi[B](splitEpi: SplitEpi[A, B])(implicit ev: Monad[F]): Reuse[ViewOptF[F, B]] =
-      self.zoom(splitEpi.get)(splitEpi.modify)
-  }
+extension [F[_], A](self: Reuse[ViewOptF[F, A]])
+  @targetName("zoomSplitEpiOpt")
```

Comment:
Why the `@targetName`? I don't think that name can be used from within scala.

Reply:
Without the `@targetName` it doesn't compile. The compiler reports a name conflict (and suggests using `@targetName`).

Reply:
Hmmm. It must be due to type erasure of Reuse.

Reply:
Yup, I think that's it.